Framework/PythonInterface/mantid/simpleapi.py

```diff
@@ -302,6 +302,39 @@ def fitting_algorithm(f):
     When applied to a function definition this decorator replaces its code with code of
     function 'wrapper' defined below.
     """
+    def gather_fitting_returns(function_name, lhs, algm):
```

Comment:
Yes, it would be nice if a `dict()` or similar was returned instead of `tuple()`. However, things are as they are and I am a bit concerned about these Fit specific changes to simpleapi, and was just wondering if the same effect could be reached by changing `Fit.cpp` only.